src/js/post-excerpt/panel.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * WordPress dependencies
+ */
+const { __ } = wp.i18n;
+const { Button, ExternalLink, TextareaControl } = wp.components;
+const { withSelect, withDispatch } = wp.data;
+const { compose } = wp.compose;
+
+/**
+ * Internal dependencies
+ */
+import { handleClick } from '../helpers';
+
+function PostExcerpt( { excerpt, onUpdateExcerpt } ) {
+	const { select } = wp.data;
+	const postId = select( 'core/editor' ).getCurrentPostId();
+	const buttonText = __( 'Generate excerpt', 'classifai' );
+
+	return (
+		<div className="editor-post-excerpt">
+			<TextareaControl
+				__nextHasNoMarginBottom
+				label={ __( 'Write an excerpt (optional)' ) }
+				className="editor-post-excerpt__textarea"
+				onChange={ ( value ) => onUpdateExcerpt( value ) }
+				value={ excerpt }
+			/>
+			<ExternalLink
+				href={ __(
+					'https://wordpress.org/support/article/settings-sidebar/#excerpt'
+				) }
+			>
+				{ __( 'Learn more about manual excerpts' ) }
```

Comment:
@dkotter can we add a text domain to the function? and other places too

Reply:
This code as copied over from Gutenberg (see https://github.com/WordPress/gutenberg/blob/4b4c4befb34d815634b85cbee23cad169ab0e073/packages/editor/src/components/post-excerpt/index.js) and as such, I've left off the text domain on these strings. This means we don't have to maintain translations for these strings as they will load from WordPress Core instead. See:

![Screen Shot 2023-03-09 at 8 17 04 AM](https://user-images.githubusercontent.com/916738/224068802-b7c19646-7ece-4d8e-97b7-f1939312c89b.png)